ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactEditText.java

```diff
@@ -102,7 +103,23 @@ public ReactEditText(Context context) {
   // TODO: t6408636 verify if we should schedule a layout after a View does a requestLayout()
   @Override
   public boolean isLayoutRequested() {
-    return false;
+    // If we are watching and updating container height based on content size
```

Comment:
If you return `true` here, in the case of single-line text, the cursor will go out of view after you've typed after the end of the visible line. That means that `onContentSizeChange` should be limited to only work with multiline texts.